pos_restricted_customer_list/__openerp__.py

```diff
@@ -1,20 +1,24 @@
 # -*- coding: utf-8 -*-
-# © 2017 Therp BV <https://therp.nl>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copyright 2017-2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
 {
-    'name': 'POS with limited list of customers downloaded',
-    'version': '8.0.0.1.0',
-    'category': 'Point Of Sale',
-    'summary': 'Optimise load time for POS where customer data not needed',
-    'author': 'Therp BV, Odoo Community Association (OCA)',
-    'website': 'https://therp.nl',
-    'license': 'AGPL-3',
+    'name': "POS: restricted customer list",
+    'version': '9.0.1.0.0',
+    'development_status': "Production/Stable",
+    'category': "Point Of Sale"'',
+    'summary': "This module will limit the download of customer data to "
+                "only those customers where this has been specifically "
+                "requested.",
+    'author': "Therp BV, Odoo Community Association (OCA)",
+    'website': "https://github.com/OCA/pos/tree/9.0/"
```

Comment:
https://github.com/OCA/pos should be enough

Reply:
It would be enough, indeed. I just chose the other way suggested by the [OCA guidelines](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11).

Reply:
Ok :+1: I prefer the shorter anyway as it's a line you don't have to worry about in future migrations